--- language/grammar/surface.ebnf	2026-08-25 00:39:43
+++ /tmp/pos-surface.ebnf	2026-09-19 00:00:00
@@ -153,8 +153,9 @@
                  | set_comprehension
                  | record_expr
                  | range_expr
                  | binder_expr
                  | match_expression
-                 | "(" , expression , ")" ;
+                 | "(" , expression , ")"
+                 | "⟥" ;
 
 tuple_expr       = "(" , expression , "," , [ expression , { "," , expression } ] , ")" ;
